Dockerfile

```diff
@@ -12,6 +12,8 @@ ENV CG_SQL_DATABASE_URI="sqlite:///:memory:"
 ENV CG_ENABLE_ADMIN="1"
 ENV CG_SECRET_KEY="key"
 
+ENV ALLOWED_ORIGINS=["https://some_site.com"]
```

Comment:
How does this work? Should it not be list of allowed origin and not a mock one? I know very little about cors

Reply:
https://github.com/Clinical-Genomics/atlas/pull/1481